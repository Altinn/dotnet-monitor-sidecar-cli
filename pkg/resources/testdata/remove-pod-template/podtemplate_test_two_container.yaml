metadata:
  annotations:
    dev.local/dd-added: "true"
    dev.local/dd-apply: '{"containerToDebug":"test","debugContainerName":"debug","tmpdirAdded":false,"secretMount":"secret"}'
  creationTimestamp: null
  name: test
spec:
  containers:
  - name: test
    resources: {}
    volumeMounts:
    - mountPath: /tmp
      name: test
  - name: test2
    resources: {}
  - args:
    - --urls
    - http://*:52323
    image: test:latest
    imagePullPolicy: IfNotPresent
    name: debug
    ports:
    - containerPort: 52323
    resources:
      limits:
        cpu: 250m
        memory: 256Mi
      requests:
        cpu: 50m
        memory: 32Mi
    securityContext:
      capabilities:
        add:
        - SYS_PTRACE
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /tmp
      name: test
    - mountPath: /etc/dotnet-monitor
      name: secret
  volumes:
  - emptyDir: {}
    name: test
  - name: secret
    secret:
      secretName: test
