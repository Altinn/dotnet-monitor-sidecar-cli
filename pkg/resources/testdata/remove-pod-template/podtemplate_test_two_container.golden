metadata:
  creationTimestamp: null
  name: test
spec:
  containers:
  - name: test
    resources: {}
    volumeMounts:
    - mountPath: /tmp
      name: test
  - name: test2
    resources: {}
  volumes:
  - emptyDir: {}
    name: test
